db/items.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE items(id INTEGER PRIMARY KEY,name TEXT,category INTEGER,image_name TEXT);
```

Comment:
```suggestion
CREATE TABLE items(id INTEGER PRIMARY KEY,name TEXT,category_id INTEGER,image_name TEXT);
```

It's a common convention to name a foreign key as `..._id`.